extensions/mentions/less/forum.less

```diff
@@ -108,6 +109,9 @@
     color: @body-bg;
   }
 }
+.Badge--group {
+  box-shadow: none;
+}
```

Comment:
Is this meant to remove shadows within autocomplete? if so, I prefer a more context oriented approach like
```less
.MentionsDropdown .Badge {...}
```